### PR TITLE
More protobuf structures

### PR DESCRIPTION
**Pull request by BAndysc**
Added parsing CMSG_GAME_OBJ_USE, CMSG_QUEST_GIVER_CHOOSE_REWARD and CMSG_MOVE to protobuf parsing

### CODE REVIEW

File: WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs
Comment (mdX7) [inline WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:447]: not sure if we should do it like this; you changed the sign here

Comment (BAndysc) [reply WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:447]: Indeed, in most handlers quest id was read as uint, hence the common structure has uint, and I've thought it probably should be uint here. I guess it doesn't matter in case of quest id, because no quest id is (and never will be) higher than max int32. But I can revert this change and just cast the value from int to uint.

Comment (mdX7) [reply WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs:447]: i'd stick to client sign and cast for such purposes, would suggest to wait on another opinion though

Comment (BAndysc) [reply WowPacketParserModule.V9_0_1_36216/Parsers/QuestHandler.cs]: Fixed the sign

**Comment by jackpoz** [comment]
What's the status of this PR ?

**Comment by BAndysc** [comment]
@jackpoz Ready to merge if other approves the change in generic CMSG_QUEST_GIVER_CHOOSE_REWARD handler

**Comment by funjoker** [comment]
bo clue about MoP, so for me it's okay

**Comment by mdX7** [comment]
> 
> 
> bo clue about MoP, so for me it's okay

agreed

**Comment by Aokromes** [comment]
maybe skyfires' wpp can give a light?

**Review by Shauren — APPROVED** [review]

**Pull request merged**
